pkg/services/object/get/exec.go

```diff
@@ -198,6 +206,44 @@ func (exec *execCtx) getChild(id oid.ID, rng *objectSDK.Range, withHdr bool) (*o
 	return child, ok
 }
 
+// copyChild fetches child object payload and streams it directly into current exec writer.
+// Returns if child header was received and if full payload was successfully written.
+func (exec *execCtx) copyChild(id oid.ID, rng *objectSDK.Range, withHdr bool) (bool, bool) {
```

Comment:
how about `getChild` method above? Is there anything stopping u from converting it into streams?